src/App.tsx

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  useEffect(() => (
+    document.addEventListener('keyup', (event) => setPressedKey(event.key))
+  ));
```

Comment:
don't forget to cleanup before component unmounts - a callback after `return` statement

```suggestion
const keyupHandler = (event) => setPressedKey(event.key);
  useEffect(() => (
    document.addEventListener('keyup', keyupHandler)
    
    return () => document.removeEventListener('keyup', keyupHandler);
  ), []);
```